modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/history/task/HistoricTaskInstanceCollectionResource.java

```diff
@@ -225,6 +226,10 @@
             queryRequest.setIncludeTaskLocalVariables(Boolean.valueOf(allRequestParams.get("includeTaskLocalVariables")));
         }
 
+        if (requestParams.containsKey("includeProcessVariables")) {
```

Comment:
Shouldn't `requestParams` be `allRequestParams` instead?

Reply:
Indeed 2230030567c3621503a19ff5cfe6eb5f30b56876